utils/utils.js

```diff
@@ -35,6 +35,7 @@ exports.scanMessage = (msg, client) => {
      const askeddiscord = new Set();
      const askedvote = new Set();
      const askedwebsite = new Set();
+     const askreddit = new set();
```

Comment:
Error here. Check whats wrong. I'd recommend you check the script works in a dummy server / or using another bot in the same server .. use #discord-logs for all the testing.....before sending here.